docs/alexa_skills_kit_configuration.md

```diff
@@ -22,7 +22,7 @@ From your [Skill's list](https://developer.amazon.com/edw/home.html#/skills/list
 
 Make sure Account linking is set to "No" and on the SSL Certificate step select "My development endpoint is a subdomain of a domain that has a wildcard certificate from a certificate authority".
 
-__ Note__ To help generate your configuration URL, you can also use [$ bst proxy urlgen](/commands/proxy.html#bst-proxy-urlgen)
+__ Note__ To help generate your configuration URL, you can also use [bst proxy urlgen](/en/latest/commands/proxy.html#bst-proxy-urlgen)
```

Comment:
I am very much not happy about this either.  

The problem I ran into was that building them locally would not translate to what RTD ultimately does. Building locally /commands/proxy.html works but then it is broken when read the docs puts it up because it needs to be /en/latest/commands/proxy.html.

If we still want to use Sphinx to build Markdown, I think the solution is to flatten everything out, do not use subdirectories in the `/docs` folder.  That way a link like `proxy.html` just works (note it doesn't have a leading `/`) for building locally and on read the docs. 

If we want to use subdirectories, which I like, and continue to use markdown, we might need to switch to MkDocs for compiling the docs, which I think has better support for these links.

I will try out MkDocs.